services.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: Consulting
+description: "Consulting services"
+---
+
+## Problems We Solve
+
+You should consider hiring us if:
+
+- You are hitting a plateau in getting your AI to do what you want.
+- Your LLMs need to be cheaper or faster.
+- You are overwhelmed by tools & frameworks.
+- You feel lost on what to prioritize and what experiments you should run.
+- You need outside expertise to evaluate your needs and vet potential talent.
+
+## Services
+
+We offer two types of services:
+
+### 1. Strategic Consulting
+
+We will save your engineering team time and money by steering them away from common pitfalls, selecting the best tools, and walking them through an approach to building AI products. Furthermore, we will introduce you to talent, vendors and partners to accelerate your goals.  
+
+### 2. Comprehensive Consulting
+
+Everything in strategic consulting, plus we build and deploy components of AI products for you.  We will work with your team to understand your needs, build a roadmap, and execute on it.
+
+## Pricing
+
+The cost of our services depends on the scope of the project.  However, the following are typical prices for our services:
```

Comment:
happy to say 

"lead by jason and hamel, with access to other experts in the field, from open source maintainers and high level operators that are building with ai"